.coafile

```diff
@@ -48,6 +48,7 @@ body_enforce_issue_reference = False
 enabled = False
 bears = LineCountBear
 max_lines_per_file = 1000
+ignore_files_regex = ^.*__init__.py$, ^.*\.gitkeep$, ^.*\.gitignore$
```

Comment:
Are you sure you need to ignore init files, coz IMO they have some code written in them.

Reply:
Yeah ik but not all init files are empty you know ;)

Reply:
@newbazz But I don't ignore all init.py files :) I will ignore it just in special case, it means when number of lines in init file is smaller than allowed minimum set. Then ignore file, means that for this type of file, I can't write the right message, because init can have less lines than set minimum.

Reply:
I don't understand this change too, is the `LineCountBear` triggered in our repo?

Reply:
Also, we don't have an `ignore_files_regex` setting :O